perun-core/src/main/java/cz/metacentrum/perun/core/impl/AttributesManagerImpl.java

```diff
@@ -2731,7 +2731,7 @@ public Attribute fillAttribute(PerunSession sess, Resource resource, Attribute a
 		//Use attributes module
 		ResourceAttributesModuleImplApi attributeModule = getResourceAttributeModule(sess, attribute);
 		if (attributeModule == null) {
-			log.debug("fillAttribute - There's no attribute module for this attribute. Attribute wasn't filled. Attribute={}", attribute);
+			log.debug("fillAttribute - There's no attribute module for this attribute. Attribute {} wasn't filled.", attribute);
```

Comment:
Original message is not ok, but having placeholder for attribute object at the end has meaning. Since ``toString()`` on Attribute if pretty long, you won't see "wasn't filled." on first sight. Maybe rephrase whole message, like: ``Attribute wasn't filled. There is no module for: {}`` and place attribute object at the end. Also I think that attribute name is sufficient in this message, so you don't have to pass the whole object.